app/views/date_of_marriage.scala.html

```diff
@@ -26,39 +26,39 @@
 formPartialRetriever: uk.gov.hmrc.play.partials.FormPartialRetriever)
 
 @templates.tamc_main(
-title = Messages("title.application.pattern", Messages("title.date-of-marriage")),
+title = messages("title.application.pattern", messages("title.date-of-marriage")),
 mainConfig = views.helpers.MainConfig(
 maybeMainDataAttributes = Some(Html(s"data-journey=${TextGenerators.formPageDataJourney("marriage-allowance:stage:transfer", marriageForm)}")))
 ){
 
-<a href="@controllers.routes.TransferController.transfer" class="link-back">@Html(Messages("generic.back"))</a>
+<a href="@controllers.routes.TransferController.transfer" class="link-back">@Html(messages("generic.back"))</a>
```

Comment:
You might be able to do away with Html here

Reply:
No test for this to make sure everything will be fine, and I don't have time now to do, Will ignore this